script/download_rust_deps.py

```diff
@@ -124,6 +124,29 @@ def make_standalone_toolchain_for_android():
     fp.close()
 
 
+def get_linux_target(target_arch):
+    return {
+        'arm64': 'aarch64-linux',
+        'x86_64': 'x86_64-linux',
+    }[target_arch]
+
+
+def get_linux_linker(target_arch):
+    return get_linux_target(target_arch) + "-gnu-gcc"
+
+
+def make_rustup_config_for_linux():
+    config_path = os.path.join(RUSTUP_HOME, 'config')
+    fp = open(config_path, "w+")
+
+    for target_arch in ['arm64', 'x86_64']:
+        # Add target to rustup config
+        fp.write("[target." + get_linux_target(target_arch) + "]\r\n")
+        fp.write("linker = \"" + get_linux_linker(target_arch) + "\"\r\n")
```

Comment:
Is the `\r` needed (here and elsewhere in the script)?

If we're using python3, this line could be
```python
fp.write(f'linker = "{get_linux_linker(target_arch)}"\n')
```
Reads a bit better, I think. This could be applied elsewhere in the script.

I also tend to define auxiliary functions used in only one function inside that function. Though that's a matter of preference here.

Otherwise, at least at this point, I don't have much else to add to this pr.